docs/source/test_network.md

```diff
@@ -32,7 +32,7 @@ the correct version tag. The command also installs the Hyperledger Fabric platfo
 binaries and config files for the version into the `/bin` and `/config` directories of `fabric-samples`.
 
```

Comment:
I did not know this was here. It would be better to link to the main install script.